docs/cookbook/extending-product-list.md

```diff
@@ -66,9 +66,123 @@ class ListedProductView extends BaseListedProductView
 }
 ```
 
-### 2. Extend `ListedProductViewFactory` so it returns the new required data
+### 2. Add new attribute to ElasticSearch
```

Comment:
In all existing articles we write it like `Elasticsearch` (lowercased s).
If you feel it should be different, please raise an issue or PR to change it everywhere. Or unify it in this PR :)